build.gradle

```diff
@@ -57,8 +57,8 @@ task runLangtonAntSergio(dependsOn: 'classes', type: JavaExec) {
     main = 'org.fundacionjala.at15.katas.langtonant.sergio.Main'
     classpath = sourceSets.main.runtimeClasspath
 }
-task adrianExample(dependsOn: 'classes', type: JavaExec) {
-    main = 'org.fundacionjala.at15.katas.langtonant.adrian.MainAdrian'
+task runLangtonAntDaniel (dependsOn: 'classes', type: JavaExec) {
```

Comment:
Hi @edb801 , please keep both tasks, yours and the one for Adrian

Reply:
hello jose i corrected it

Reply:
Thanks Daniel, looks like accidently, the line below was removed from Adrian's task:
```
classpath = sourceSets.main.runtimeClasspath
```
Without that, Adrian's task would be broken, please restore that change in a new commit